posts/demos/_posts/2021-05-28-erasing.md

```diff
@@ -106,25 +106,12 @@ if(eraser) {
 ```
 
 ### Erasing and `Canvas#backgroundColor`, `Canvas#overlayColor`
-**If you're using `backgroundColor | overlayColor` and erasing you should read this carefully.**
+Erasing uses `fabric.Object#clipPath` under the hood.
 
-To achieve erasing, `backgroundColor | overlayColor` must be fabric objects (see **Erasing and the `clipPath`**).
-Because of this, the mixin extends `fabric.StaticCanvas`. 
-This may cause issues when resizing the canvas. You might need to resize `backgroundColor | overlayColor` as well.
+`backgroundColor` and `overlayColor` don't extend `fabric.Object`, hence they can't be erased.
 
-```js
-canvas.setOverlayColor('rgba(0,0,255,0.4)');
-canvas.overlayColor instanceof fabric.Rect  //  equals to true;
-```
-
-convenice for setting color and `erasable` property:
+A simple workaround would be adding a rect at the bottom/top of the object stack that could behave as `backgroundColor` or `overlayColor`.
```

Comment:
what is wrong with using a rect as backgroundImage?

Reply:
You mean for the bgImage to act as bgColor?

Reply:
Today on backgroundImage you can set any fabric Object.
In that case you can add also a rect, if you do not need an image too, to make it behave as a color

Reply:
Good workaround

Reply:
In fact you could add a `Group` and do a lot of thing if you listen to `erasing:end`